arrows/core/video_input_image_list.h

```diff
@@ -36,15 +37,18 @@
 #include <arrows/core/kwiver_algo_core_export.h>
 
 namespace kwiver {
+
 namespace arrows {
+
 namespace core {
 
-/// Video input using list of images.
-// ----------------------------------------------------------------
+// ----------------------------------------------------------------------------
 /**
- * This class implements a video input algorithm using a list of
- * images to simulate a video. Only the images are returned. This
- * algorithm produces no metadata.
+ * @brief Video input using list of images.
```

Comment:
Can we also be consistent on `@brief` vs `\brief` within this file?  I'd suggest `\brief`.

Reply:
Sure (sorry, my bad). I was going to say something about `@cmd` being more common, but despite that I feel confident I've *seen* `@cmd` much more often than `\cmd`, git says that impression is incorrect. Which WFM, as I prefer `\cmd` anyway :slightly_smiling_face:.